src/Snippets/Docs/Chaos.Behavior.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using Polly.Retry;
+using Polly.Simmy;
+using Polly.Simmy.Behavior;
+
+namespace Snippets.Docs;
+
+internal static partial class Chaos
+{
+    public static void BehaviorUsage()
+    {
+        static ValueTask RestartRedisVM() => ValueTask.CompletedTask;
+
+        #region chaos-behavior-usage
+        // Behavior using the default options.
+        // See https://www.pollydocs.org/chaos/behavior#defaults for defaults.
+        var optionsDefault = new BehaviorStrategyOptions();
+
+        // To use a custom function to generate the behavior to inject.
+        var optionsWithBehaviorGenerator = new BehaviorStrategyOptions
+        {
+            BehaviorAction = static args => RestartRedisVM(),
+            Enabled = true,
+            InjectionRate = 0.6
+        };
+
+        // To get notifications when a behavior is injected
+        var optionsOnBehaviorInjected = new BehaviorStrategyOptions
+        {
+            BehaviorAction = static args => RestartRedisVM(),
+            Enabled = true,
+            InjectionRate = 0.6,
+            OnBehaviorInjected = static args =>
+            {
+                Console.WriteLine("OnBehaviorInjected, Operation: {0}.", args.Context.OperationKey);
+                return default;
+            }
+        };
+
+        // Add a behavior strategy with a BehaviorStrategyOptions{<TResult>} instance to the pipeline
```

Comment:
```suggestion
        // Add a behavior strategy with a BehaviorStrategyOptions instance to the pipeline
```

no generic variant available

Reply:
Done